packages/node/src/indexer/project.service.ts

```diff
@@ -388,7 +392,7 @@ export class ProjectService {
     );
   }
 
-  private async reindex(targetBlockHeight: number): Promise<void> {
+  async reindex(targetBlockHeight: number): Promise<void> {
```

Comment:
The subcommands PR has moved this to another location. Will require rebasing.

Reply:
Sure, once reindex pr is merged, I will have this updated